contracts/account/ArgentPluginAccount.cairo

```diff
@@ -185,6 +185,26 @@ func __validate_declare__{
     return ();
 }
 
+@external
+func __validate_deploy__{
+    syscall_ptr: felt*,
+    pedersen_ptr: HashBuiltin*,
+    ecdsa_ptr: SignatureBuiltin*,
+    range_check_ptr
+} (
+    class_hash: felt,
+    ctr_args_len: felt,
+    ctr_args: felt*,
+    salt: felt
+) {
+    alloc_locals;
+    // get the tx info
+    let (tx_info) = get_tx_info();
+    // validate the signer signature only
+    ArgentModel.validate_signer_signature(tx_info.transaction_hash, tx_info.signature_len, tx_info.signature);
```

Comment:
i think this should also validate the guardian signature, like ArgentAccount

Reply:
indeed, I forgot to update the `ArgentPluginAccount` contract.